streamlit_drawable_canvas/frontend/src/lib/circle.ts

```diff
@@ -22,6 +23,7 @@ class CircleTool extends FabricTool {
     this.strokeWidth = strokeWidth
     this.strokeColor = strokeColor
     this.fillColor = fillColor
+    this.minSize = strokeWidth
```

Comment:
I don't think this is going to be used outside the class right? We can make it private if that's the case, by prepending with an underscore, so `this._minsize` and `_minsize: number` in the declaration

The other arguments are shared between multiple classes, which is why they are public.

Also if we make this private, we can call it even more explicitly `_minRadius` instead :wink: